.github/renovate.json

```diff
@@ -6,5 +6,8 @@
   "ignoreDeps": [
     "pnpm",
     "node"
+  ],
+  "labels": [
+    "renovate"
```

Comment:
wouldn't `dependencies` be more generic?

Reply:
yes